### PR TITLE
Add maxtext nightly test on 1 slice of v4-8

### PR DESCRIPTION
**Pull request by tonyjohnchen**
# Description

Please include a summary of relevant context/issue and your changes.

Run maxtext nightly on 1 slice of v4-8

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...

**List links for your tests (use go/shortn-gen for any internal link):** ...

Test logs: https://screenshot.googleplex.com/KNCMu8sQyxeNXWs.png
Test result: https://screenshot.googleplex.com/7sXcqLTgiAKcXKc.png

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: configs/maxtext/maxtext_gce_config.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+def get_maxtext_nightly_config(
+    tpu_version: TpuVersion,
+    tpu_cores: int,
+    tpu_zone: str,
+    time_out_in_min: int,
+    project_name: str = PROJECT_NAME,
+    runtime_version: str = RUNTIME_IMAGE,
+    network: str = "default",
+    subnetwork: str = "default",
+    resolution: int = 512,
```
Comment (RissyRan) [inline configs/maxtext/maxtext_gce_config.py:38]: Do you need this `resolution` param or `extraFlags`? I add always add them back when you need them in the future.

**Review by RissyRan — COMMENTED** [review]
Just one comment, others look good to me. Thanks!

File: configs/maxtext/common.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Google LLC
```
Comment (RissyRan) [inline configs/maxtext/common.py:1]: For the path, I am having a [PR](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/74) to add team subfolders under configs and dags. Inside of team subfolder, the code structure mainly depends on how your team organizes those tests. Probably no change is needed at this moment, and I will let you know once that's approved. Thanks!

**Review by RissyRan — APPROVED** [review]
LGTM

**Review by jonb377 — COMMENTED** [review]
LGTM, thanks Tony!

**Pull request merged**